wordBank.js

```diff
@@ -1,17 +1,19 @@
 "use strict";
 
 const wordBank = ["abcdefghijklmnopqrstuvwxyz", "dad", "brother"];
-
 const mySection = document.getElementById("word-bank");
-let currentWord = wordBank[0]; // test for whole alphabet
-let currentWordWithSpan = "";
-for (let i in currentWord) {
-  currentWordWithSpan += `<span id="${i}">${currentWord[i]}</span>`;
-}
 
-mySection.innerHTML = currentWordWithSpan;
+const renderWord = () => {
+  let currentWord = wordBank[0]; // test for whole alphabet
```

Comment:
can you make currentWord a global variable? since other methods use it, you don't have to re-define again.

Reply:
There is one issue with that. If I make it global, it won't be updated to new word when the event is triggered. So every time I call render function, it always render the original word, not the new word.